package.json

```diff
@@ -129,12 +129,12 @@
     "@angular/cli": "^14.2.3",
     "@angular/compiler-cli": "^14.2.3",
     "@angular/language-service": "^14.2.3",
+    "@babel/runtime": "^7.18.9",
     "@ngrx/store-devtools": "^14.3.0",
     "@nguniversal/builders": "^14.2.0",
     "@sapui5/ts-types-esm": "1.98.0",
```

Comment:
not related to this file, but `scripts/package.json` has be a dangling/dead config for a while. File for tsconfig existed in the past, which can be found here https://github.com/SAP/spartacus/pull/4485/files, but was no longer used. Especially since we deleted the .spec file, we do not need it anyhow

Reply:
Was thinking the same, but wasn't _bold_ enough. Done!